_posts/blog/2016-12-15-running-on-fedora-25.md

```diff
@@ -0,0 +1,327 @@
+---
+title: "Running Accumulo on Fedora 25"
+author: Christopher Tubbs and Mike Miller
+reviewers: 
+---
+
+Apache Accumulo has been available in [Fedora] for the last few releases.
+Recently, the [maintainers] have updated it to version `1.6.6` and have made
+some improvements to the packaging and launch scripts to provide a good
+out-of-box experience. This post will discuss the basic setup procedures for
+running Accumulo in the latest version, `Fedora 25`.
+
+## Prepare the system
+
+The Fedora maintainers provide a meta-package to install accumulo and all of
+its dependencies. We also recommend installing the JDK, so you'll have access
+to the `jps` command, and `tuned` for setting system performance tuning
+parameters from a profile. We'll also make sure our preferred editor is
+installed (replace `vim` with your preferred editor):
+
+```bash
+sudo dnf install accumulo java-1.8.0-openjdk-devel tuned vim
+```
+
+It is possible to install only a specific service, if desired, but it won't
+hurt to have extra packages installed and not used. For the single-node setup,
+we'll need almost everything. For the multi-node setup, it might make more
+sense to be selective about which you choose to install on each node.
+
+To set up `tuned`, do:
+
+```bash
+sudo systemctl start tuned.service     # start service
+sudo tuned-adm profile network-latency # pick a good profile
+sudo tuned-adm active                  # verify the selected profile
+sudo systemctl enable tuned.service    # auto-start on reboots
+```
+
+## Running a single-node
+
+These instructions will assume you're using Fedora 25, fully up-to-date (`sudo
+dnf --refresh upgrade`).
+
+To run on a single-node, you don't need to run HDFS. Accumulo can use the local
+filesystem as a volume instead. By default, it uses `/tmp/accumulo`. Let's
+change that to something which will survive a reboot:
+
+```bash
+sudo vim /etc/accumulo/accumulo-site.xml
+```
+
+Change the value of the `instance.volumes` property from `file:///tmp/accumulo`
+to `file:///var/tmp/accumulo` in the configuration file (or another preferred
+location).
+
+While you are editing the Accumulo configuration file, you should also change
+the default `instance.secret` from `DEFAULT` to something else. You can also
+change the credentials used by the `tracer` service now, too. If you use the
+`root` user, you'll have to set its password to the same one you'll use later
+when you initialize Accumulo. If you use another user name, you'll have to
+create that user later.
+
+Now, Hadoop's default local filesystem handler isn't very good at ensuring
```

Comment:
Why tell users to configure Hadoop if you they are setting `instance.volumes` to file system above?